docs/src/usage/scenario_generation.md

```diff
@@ -115,7 +115,24 @@ ADRIA.fix_factor!(dom;
     coral_cover_tol=1.0
 )
 ```
+The ranges for sampling can also be constrained if running scenarios for a particular 
+range of a parameter is of interest. For example, if one wanted to investigate only 
+scenarios with high fogging and seeding, the following example could be used:
 
+```julia
+dom = ADRIA.load_domain()
+
+# Adjust seeding bounds. Note only lower and upper bounds needed because it is a 
+# uniform distribution.
+ADRIA.set_factor_bounds!(dom, :N_seed_TA, (500000.0, 1000000.0 + 1.0))
+ADRIA.set_factor_bounds!(dom, :N_seed_CA, (500000.0, 1000000.0 + 1.0))
+ADRIA.set_factor_bounds!(dom, :N_seed_SA, (500000.0, 1000000.0 + 1.0))
```

Comment:
Any way of setting bounds for a group of factors like in the `fix_factor` example above?

Reply:
There is a function for this but I just added an example to the markdown